crypto/crypto.go

```diff
@@ -62,3 +73,46 @@ func FromECDSAPub(pub *ecdsa.PublicKey) []byte {
 func S256() elliptic.Curve {
 	return secp256k1.S256()
 }
+
+// LoadECDSAFromString create ecdsa privatekey from string
+// ecStr should start with 0x or 0X
+func LoadECDSAFromString(ecStr string) (*ecdsa.PrivateKey, error) {
+	if len(ecStr) >= 2 && ecStr[0] == '0' && (ecStr[1] == 'x' || ecStr[1] == 'X') {
+		key, err := hex.DecodeString(ecStr[2:])
+		if err != nil {
+			return nil, err
+		}
+		return ToECDSA(key)
+	}
+	return nil, errors.New("Input string not a valid ecdsa string")
```

Comment:
Suggest to define all the errors used in this file at the beginning as const variables.

Reply:
en